src/modules/esl-carousel/core/esl-carousel.less

```diff
@@ -50,5 +50,6 @@ esl-carousel {
 }
 
 [esl-carousel-container] {
+  --esl-carousel-clip-shift: 5px;
```

Comment:
What does it mean - clip-shift? I can't understand.
Maybe should name it `esl-carousel-arrow-shift`?

Reply:
It's not related to the arrow, its the area around the carousel where overflow:clip does not have effect.
So we see how slides move and that area, an d also we use such area to display slide decorations like shadow, etc (otherwise overflow:clip will cut it of)

But agree that the variable name is not an ideal one, so any ideas welcome

Reply:
I see. Maybe it should be `esl-carousel-side-margin`?

Reply:
Current vote for `esl-carousel-side-space`

Vote history:
esl-carousel-
  - side-space 1 1 1.5 1.5 2 1.5
  - side-offset 1 1 1.5 1.5 1.5
  - space-around 1 1.5 1